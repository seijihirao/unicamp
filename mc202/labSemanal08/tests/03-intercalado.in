inserir 6122
inserir 3937
inserir 8924
em-ordem
inserir 2553
pre-ordem
inserir 5031
pos-ordem
inserir 5897
excluir 5031
excluir 2553
excluir 3937
excluir 6122
excluir 8924
excluir 5897
largura
terminar
