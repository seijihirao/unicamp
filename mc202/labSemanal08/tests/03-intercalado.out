3937 6122 8924 
6122 3937 2553 8924 
2553 5031 3937 8924 6122 
vazia
